README.md

```diff
@@ -60,3 +60,18 @@ For accessibilit testing:
 ```bash
 $ yarn test:a11y
 ```
+For code coverage:
+```bash
+$ yarn test:coverage
+$ yarn sonar-scanner
+```
+## Troubleshooting
+
+### nyc warings while running  ```yarn install``` on yarn version 1.0.1
+
+This is currently an Open issue : https://github.com/yarnpkg/yarn/issues/3751
```

Comment:
There is currently an open issue,
also include a short example of the warnings only one or two lines